mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/HadoopFileSystemBlobStore.java

```diff
@@ -41,22 +39,14 @@ public class HadoopFileSystemBlobStore implements BlobStore {
 
     private final File localStoreDir;
 
-    final Striped<Lock> locks = Striped.lock(1024);
```

Comment:
is it possible now without lock two threads can try downloading the target zip from s3 same time?